### DISCUSSION TITLE
Hoping to add a hook about form

### DISCUSSION THREAD

**Discussion by 332lfgb6** [remix-run/react-router#11348] [Proposals]
When using useFetch to submit the form, if `<fetch.Form action=''>`, you can fetch the return values of `loader` and `action` in `useLoaderData` and `useActionData`. If  `<fetch.Form action='xxx'>`, The returned values in `loader` and `action` cannot be obtained

**Comment by kiliman** [comment]
If you're using `useFetcher`, then the returned data is on `fetcher.data`

https://reactrouter.com/en/main/hooks/use-fetcher#fetcherdata
